### ISSUE TITLE
Bookmark all tabs repositioning / placing

### ISSUE THREAD

**Issue by zputnyq** [bromite/bromite#943]
Bromite 88.0.4324.187

### Is your feature request related to privacy?

No

### Is there a patch available for this feature somewhere?

Not sure

### Describe the solution you would like

As its subject stated, it is related to the bookmark all tabs option position/place.

### Describe alternatives you have considered

In my opinion bookmark all tabs migth be better placed above or under bookmark, it will looks well arrange. It might also avoid mistapping of exit option.

**Comment by csagan5** [comment]
Fixed in `88.0.4324.207`.

**Comment by elcste** [comment]
I agree this change makes sense. "Bookmark all tabs" was funny by itself at the bottom.

But I have a phone with a tiny screen, and not all menu items are visible without scrolling. The menu item I use more than any, "Find in page", had been the last visible item, but with this change it's pushed down so I have to scroll to use it.

I don't have any good compromise suggestions :-/ but maybe someone else does?

**Comment by tp0** [comment]
The original place at the bottom was odd, but now it is between "Bookmarks" and "Recent apps", which are very commonly used and it feels like it is too easy to accidentally tap the "Bookmark all tabs" (which is rather annoying, as you can't cancel the action).

Should it be in the regular 3-dot menu at all? It is accessible in the other 3-dot menu when the tab switching view is activated. That seems like a more natural place for it.
